e2e/utils/create-project-utils.ts

```diff
@@ -39,7 +39,7 @@ export function newProject({
   packageManager = getSelectedPackageManager(),
 } = {}): string {
   try {
-    const useBackupProject = packageManager !== 'pnpm';
+    const useBackupProject = /*packageManager !== 'pnpm'*/ true;
```

Comment:
should just get rid of the flag in this case.

Reply:
True, I left it for testing. Let me remove it quickly